Sun_Shengran/progress.md

```diff
@@ -3,8 +3,8 @@
 **Project: Recycle Bin**
 
 Overall Direction:
-- Build a recovery page for instructors to display user deleted courses, sessions and students
-- Allow instructors to recover deleted courses, sessions and students from the recycle bin
+- Build a recovery page for instructors to display user deleted courses and sessions (whose corresponding courses are not deleted)
+- Allow instructors to recover selected/all courses and sessions from the recycle bin
```

Comment:
In light of your exchange with the Prof, you might also want to make clear the fact that students' details will also be restored when a session is restored (although this is mentioned in the details below)